File: Java/POOber/src/poober/Account.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package poober;

/**
 *
 * @author laura
 */
public class Account {
    
   Integer Id;
   String name;
   String document;
   String email;
   String password;
          
   //metodo costructor
   
   public void Account (String name, String document){
       this.name = name;
       this.document = document;
   }
   
    
}

File: PHP/Account.php
<?php
class Account{
    Public $id;
    Public $name;
    Public $email;
    Public $password;
    Public $document;

public function __construct ($name, $document, $email, $password){
    $this->name = $name;
    $this ->document = $document;
    $this ->email = $email;
    $this ->password = $password;


}

public function printDataAccount (){
    echo ("impresion de nombre ".$this ->$name);
}
}
?>